layer/nrlf/core/tests/test_validators.py

```diff
@@ -722,6 +722,40 @@ def test_validate_ssp_content_without_any_context_related():
 
 
 def test_validate_ssp_content_without_asid_in_context_related():
```

Comment:
I don't think this test name represents the test. It's called `test_validate_ssp_content_without_asid_in_context_related()` but the docref in the test doesn't have ssp content and does have an ASID but it's invalid.

Reply:
i got the test names the wrong way round - oops ( the joys of copy paste)